client/tests/integration/smartcontracts/executor_with_admin/src/lib.rs

```diff
@@ -2,172 +2,34 @@
 //! If authority is not `admin@admin` then default validation is used as a backup.
 
 #![no_std]
-#![allow(missing_docs, clippy::missing_errors_doc)]
 
 #[cfg(not(test))]
 extern crate panic_halt;
 
-use iroha_executor::{
-    data_model::evaluate::{EvaluationError, ExpressionEvaluator},
-    parse,
-    prelude::*,
-    smart_contract,
-};
+use iroha_executor::{parse, prelude::*, smart_contract};
 use lol_alloc::{FreeListAllocator, LockedAllocator};
 
 #[global_allocator]
 static ALLOC: LockedAllocator<FreeListAllocator> = LockedAllocator::new(FreeListAllocator::new());
 
+#[derive(Constructor, ValidateEntrypoints, ExpressionEvaluator, Validate, Visit)]
+#[visit(omit(visit_instruction))]
 struct Executor {
     verdict: Result,
     block_height: u64,
     host: smart_contract::Host,
 }
 
-impl Executor {
-    /// Construct [`Self`]
-    pub fn new(block_height: u64) -> Self {
-        Self {
-            verdict: Ok(()),
-            block_height,
-            host: smart_contract::Host,
-        }
+#[allow(dead_code)] // Used in `Visit` derive.
```

Comment:
If so then why does it trigger the warning...?

Reply:
Having this attribute seems to be dangerous to me. It may mean that this function is really not used and macro was expanded to something else

Reply:
Same for all other validator files

Reply:
And looks like that is exactly why integration test with validator upgrade is failing